src/main/java/com/ceritoxi/util/Math.java

```diff
@@ -1,130 +1,457 @@
 package com.ceritoxi.util;
+import java.util.Stack;
 
+public class Meth {
+
+    public static boolean isDividableByXorY(int dividend, int x, int y) {
+        if ((x != 0 && dividend % x == 0) || (y != 0 && dividend % y == 0)) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfNumbersUnderZThatAreDividableByXorY (int x, int y, int z) {
+        int sum = 0;
+
+        if (z < 1) {
+            return sum;
+        }
+        for (int i = 1; i < z; i++) {
+            if (isDividableByXorY(i, x, y)) {
+                sum += i;
+            }
+        }
+        return sum;
+    }
+
+    public static boolean isEven (int number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEven (long number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfEvenFibonacciTermsUnderX (int x) {
+
+        int result = 0;
+        int sum = 0;
+        int firstTerm = 0;
+        int secondTerm = 1;
+
+        while (sum < x) {
+            if (isEven(sum)) {
+                result += sum;
+            }
+            sum = firstTerm + secondTerm;
+            firstTerm = secondTerm;
+            secondTerm = sum;
+        }
+        return result;
+    }
+
+    public static boolean isPrime (long number) {
+
+        if (number == 2 || number == 3 || number == 5) {
+            return true;
+        }
+        if (number <= 1 || number % 2 == 0 || number % 3 == 0 || number % 5 == 0) {
+            return false;
+        }
+
+        for (long i = 3; i <= java.lang.Math.sqrt(number); i += 2) {
```

Comment:
you can improve the readability by static importing the sqrt function

Add to the imports : ``` import static java.lang.Math.sqrt;``` then you can just use ```sqrt(...)``` in the code